src/tallies.cpp

```diff
@@ -302,25 +346,89 @@ void Tallies::update_avg_and_var(double x, double& x_avg, double& x_var) {
             ((x_var_old) / (dgen - 1.));
 }
 
+void make_tallies_filter(Tallies& tallies, const YAML::Node& node) {
+  if (!node["tally-filters"]) {
+    fatal_error("tally-filter are not given to evaluate the tallies.");
+  }
+
+  // position-filters
+  if (node["tally-filters"]["position-filters"]) {
```

Comment:
```suggestion
  if (node["tally-filters"]["position-filters"] && node["tally-filters"]["position-filters"].IsSequence()) {
```
We should also check to make sure that the `position-filters` entry is a list. If not, we will fatal_error down below.